From 16562b3cb150dbde28b3296a0cf30e862fc1ce74 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 25 Jan 2020 20:11:41 +0100
Subject: [PATCH 1/1] watchdog: cannot select CONFIG_CONFIG_MPC8xx_WATCHDOG

There is no configuration option CONFIG_CONFIG_MPC8xx_WATCHDOG. So we
should not try to select it.

The select statement was introduced in a patch that was meant to only sort
config options without changing them.

Fixes: b3134ffbd944 ("watchdog: Kconfig: Sort entry alphabetically")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/watchdog/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/watchdog/Kconfig b/drivers/watchdog/Kconfig
index 2b8064dfae..8b65b5a88c 100644
--- a/drivers/watchdog/Kconfig
+++ b/drivers/watchdog/Kconfig
@@ -103,7 +103,6 @@ config WDT_CDNS
 config WDT_MPC8xx
 	bool "MPC8xx watchdog timer support"
 	depends on WDT && MPC8xx
-	select CONFIG_MPC8xx_WATCHDOG
 	help
 	   Select this to enable mpc8xx watchdog timer
 
-- 
2.24.1

